### DISCUSSION TITLE
Brazil not supported?

### DISCUSSION THREAD

**Discussion by andreynering** [stackaid/stackaid#6] [Q&A] [answered]
Hi!

I got interested in Stripe and I'm taking a look at the system. Trying to signup as an OSS maintainer.

On the Stripe step I don't see "Brazil" being listed as one of the options. Is this a bug or intentional?

I remember that when I tried to signup for GitHub Sponsors I had the same issue, and had to ping GitHub's support about it (it was then added to the list).

![Screenshot 2023-01-26 at 11 03 20](https://user-images.githubusercontent.com/7011819/214855926-09a656a7-557f-4b12-9798-4c01ffa61657.png)

**Comment by wescarr** [comment]
Hey @andreynering I will reach out to Stripe to see why it's not available for Stripe Connect Express accounts. Apologies for the hassle!

**Reply by andreynering** [reply]
No hurry. Thanks!

**Comment by wescarr** [comment, marked as answer]
@andreynering I heard back from Stripe support and as I suspected Brazil is not a supported region when using Stripe Express Connect accounts. We can however generate a one time use link to setup a Standard account for you and then finish the onboarding process for you afterwards. Cheers.
